### ISSUE TITLE
Config syslinux does point to rancher

### ISSUE THREAD

**Issue by dgiebert** [burmilla/os#26]
While testing commands while migrating to Hugo:
```yaml
sudo ros config syslinux
Unable to find image 'rancher/os-console:v1.9.0-rc1' locally
system-docker: Error response from daemon: manifest for rancher/os-console:v1.9.0-rc1 not found.
See 'system-docker run --help'
```

**Comment by olljanat** [comment]
Good catch. Included fix to #25

**Comment by olljanat** [comment]
Fix confirmed on our release test version. Will close this.
